HackYourFuture-CPH/fp-class19
# Backend: Add update User endpoint

# Description

Api endpoind for editing and updating information on the User profile page

Fixes #108

# How to test?

Pull the Branch, 
Run: npm run dev ,

go to: [http://localhost:3000/api/documentation] to check swagger documentation for the endpoint,

# Checklist
